src/docs/Components/Hero/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Hero'
+order: 1
+description: 'A hero is the first thing your user sees when the page loads. This is a zone that you can use to draw the  attention of the user on a specific message.'
+status:
+  sketch: 'bêta'
+  scss: 'bêta'
+---
+
+> A hero is the first thing your user sees when the page loads. This is a zone that you can use to draw the attention of the user on a specific message.
```

Comment:
I suggest:
A hero is an element that you can use on the top of your pages add visual ambience or draw the user's attention on a specific message.

Reply:
good, thx